socorro/unittest/external/es/test_crashstorage.py

```diff
@@ -615,3 +612,215 @@ def esindex_fn(*args, **kwargs):
             body=document,
             **additional
         )
+
+    @mock.patch('socorro.external.es.connection_context.elasticsearch')
```

Comment:
Why not `@mock.patch('elasticsearch.Elasticsearch')` instead? That's what we do, for example, in so many places where we mock on `requests`. E.g. `@mock.patch('requests.get')` (instead of `@mock.patch('crashstats.crashstats.models.requests')`)